src/main/java/net/dv8tion/jda/api/managers/GuildStickerManager.java

```diff
@@ -0,0 +1,210 @@
+     * <ul>
+     *     <li>{@link #NAME}</li>
+     *     <li>{@link #DESCRIPTION}</li>
+     *     <li>{@link #TAGS}</li>
+     * </ul>
+     *
+     * @param  fields
+     *         Integer value containing the flags to reset.
+     *
+     * @return GuildStickerManager for chaining convenience
+     */
+    @Nonnull
+    @Override
+    GuildStickerManager reset(long... fields);
+
+    /**
+     * The {@link Guild} this Manager's {@link GuildSticker} is in.
+     *
+     * <p>This is null if {@link GuildSticker#getManager()} is used no a sticker with an uncached guild.
+     *
+     * @return The {@link Guild Guild}, or null if not present.
+     *
+     * @see    #getGuildId()
+     */
+    @Nullable
+    Guild getGuild();
+
+    long getGuildIdLong();
+
+    @Nonnull
+    default String getGuildId()
+    {
+        return Long.toUnsignedString(getGuildIdLong());
+    }
+
+    /**
+     * Sets the <b><u>name</u></b> of the sticker.
+     *
+     * <p>A sticker name <b>must</b> be between 2-30 characters long!
+     *
+     * <p><b>Example</b>: {@code catDance} or {@code dogWave}
+     *
+     * @param  name
+     *         The new name for the sticker (2-30 characters)
+     *
+     * @throws IllegalArgumentException
+     *         If the provided name is {@code null} or not between 2-30 characters long
+     *
+     * @return GuildStickerManager for chaining convenience
+     */
+    @Nonnull
+    @CheckReturnValue
+    GuildStickerManager setName(@Nonnull String name);
+
+
+    /**
+     * Sets the <b><u>description</u></b> of the sticker.
+     *
+     * <p>A sticker description <b>must</b> be between 2-100 characters long!
+     *
+     * @param  description
+     *         The new description for the sticker (2-100 characters)
+     *
+     * @throws IllegalArgumentException
+     *         If the provided description is {@code null} or not between 2-100 characters long
+     *
+     * @return GuildStickerManager for chaining convenience
+     */
+    @Nonnull
+    @CheckReturnValue
+    GuildStickerManager setDescription(@Nonnull String description);
+
+    /**
+     * Sets the <b><u>tags</u></b> of the sticker.
+     * <br>These are used for auto-complete when sending a message in the client, and for the sticker picker menu.
+     *
+     * <p>The combined list of sticker tags <b>must</b> at most be 200 characters long!
+     *
+     * <p><b>Example</b>: {@code catDance} or {@code dogWave}
+     *
+     * @param  tags
+     *         The new tags for the sticker (up to 200 characters)
+     *
+     * @throws IllegalArgumentException
+     *         Thrown if
+     *         <ul>
+     *             <li>{@code tags} is {@code null}</li>
+     *             <li>{@code tags} is empty</li>
+     *             <li>{@code tags} contains {@code null} or empty strings</li>
+     *             <li>The concatenated tags are more than 200 characters long (including commas between tags)</li>
+     *         </ul>
```

Comment:
Lets put the `If X` on each line as that is the pattern we're starting to see appear after we've discussed in other PRs. It reads better
```suggestion
     *         <ul>
     *             <li>If {@code tags} is {@code null}</li>
     *             <li>If {@code tags} is empty</li>
     *             <li>If {@code tags} contains {@code null} or empty strings</li>
     *             <li>If the concatenated tags are more than 200 characters long (including commas between tags)</li>
     *         </ul>
```

Applies to multiple places in the code.